apps/server/src/modules/me/api/me.uc.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { SchoolService } from '@src/modules/school';
+import { UserService } from '@src/modules/user';
+import { EntityId } from '@shared/domain/types';
+import { MeResponseMapper } from './mapper';
+import { MeResponse } from './dto';
+
+@Injectable()
+export class MeUc {
+	constructor(private readonly schoolService: SchoolService, private readonly userService: UserService) {}
+
+	// no authorization is required
```

Comment:
Den Kommentar find ich in der Form überflüssig, weil man ja im Code sieht, dass keine Authorisierung durchgeführt wird. Wenn er stehen bleiben soll, dann sollte er mMn eine Begründung enthalten.